actions/gptscript.tsx

```diff
@@ -17,8 +17,21 @@ export const parseContent = async (toolContent: string): Promise<Tool[]> => {
   ) as Tool[];
 };
 
-export const parseBlock = async (ref: string) => {
-  return await gpt().parse(ref);
+/**
+ * Verifies that a tool exists by parsing it.
+ * @param toolRef The tool reference to verify.
+ * @returns A boolean indicating whether the tool exists.
+ */
+export const verifyToolExists = async (toolRef: string) => {
+  // skip verification if the tool is a system tool
+  if (toolRef.startsWith('sys.')) return true;
```

Comment:
There is a flag builtin to the GPTScript CLI called `--list-tools` that will list all system tools. It would seem that doesn't have parity in the Node SDK. If it did exist I'd advocate for using it here. However, not a huge deal in the mean time.